src/strings/en-us.json

```diff
@@ -1625,5 +1625,38 @@
     "MachineTranslated": "Machine Translated",
     "ForeignPartsOnly": "Forced/Foreign parts only",
     "HearingImpairedShort": "HI/SDH",
-    "LabelIsHearingImpaired": "For hearing impaired (SDH)"
+    "LabelIsHearingImpaired": "For hearing impaired (SDH)",
```

Comment:
Just for future reference, having the translations in alphabetical order is preferred since it can make them easier to find and helps avoid conflicts between different PRs. It's fine to leave as is here though. :slightly_smiling_face:

Reply:
I had originally sorted the strings into the file using "Sort Lines" in VS Code but I believe there were a bunch of other strings that weren't properly sorted beforehand so it ended up sorting those too and causing frequent merge conflicts for me.